content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,6 +16,9 @@ weight: 10
 ## Security
 
 - **Added** the [`v1beta1` authorization policy model](/blog/2019/v1beta1-authorization-policy/) for enforcing access control. This will eventually replace the [`v1alpha1` RBAC policy](/docs/reference/config/security/istio.rbac.v1alpha1/).
+- **Added** experimental support for [automatic mutual TLS](docs/tasks/security/authentication/auto-mtls/).
+- **Added** experimental [DNS Certificate Management](docs/tasks/security/dns-cert/).
+- **Added** experimental support for [trust domain migration](/docs/tasks/security/authorization/authz-td-migration/).
```

Comment:
```suggestion
- **Added** (experimental) support for [trust domain migration](/docs/tasks/security/authorization/authz-td-migration/).
```

Reply:
This does not follow the pattern of the rest of the release notes